src/main/scala/org/scalajs/testsuite/utils/Platform.scala

```diff
@@ -0,0 +1,27 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite.utils
+
+import scala.scalajs.runtime
+
+object Platform {
```

Comment:
This file should not be here. It is in the `src/main/` directory, so it would be part of the published artifact.